### ISSUE TITLE
Move all public channel messages to GameMessageQueue

### ISSUE THREAD

**Issue by czeano** [czeano/upbeat#26]
Old way: 
    def public_command(player, message):
        if message == 'asdf':
            return do_asdf(player)

    def do_asdf(player):
        process_stuff()
        return "Player.Name successfully ASDFs."

New way: 

    def public_command(player, message):
        if message == 'asdf':
            do_asdf(player)

    def do_asdf():
        process_stuff()
        GameMessageQueue.add_message("Player.Name successfully ASDFs.")

**Comment by czeano** [comment]
As of Pull Request #25 the hooks should all be there to do this cleanly. Requires some edge-case testing because I forget if the public and private command injectors rely on getting messages back to know it worked.  (If they do, they need to be changed... they basically need to be asynchronous. In fact, remind me to make a separate issue that command injectors should asynchronously insert into a command queue)

**Comment by czeano** [comment]
Addendum: There should also be an error message queue. This WILL require changes to the existing interfaces -- after they Game object for public messages, they can poll for error messages and route those appropriately. For the case of the Slackbot, there should be a way to specify a channel (either public or DM) that error messages should be delivered to. For CLI, uh.... dump them to the screen I guess?

Basically there shouldn't be any print()s anywhere in the code.
